bin/watch-dev.js

```diff
@@ -21,19 +21,20 @@ async function build() {
     watch: !once,
     babelConfig: {
       plugins: [runtime, styled],
-      runtimeHelpers: true,
+      babelHelpers: 'runtime',
       include: [
-        `${__dirname}/../src/frontend/**`,
-        `${__dirname}/../src/locale/*`,
-        `${__dirname}/../src/utils/**`,
-        `${__dirname}/../src/backend/utils/view-helpers/view-helpers.ts`,
-        `${__dirname}/../src/backend/utils/filter/filter.ts`,
-        `${__dirname}/../src/backend/decorators/**`,
+        'src/frontend/**',
+        'src/locale/*',
+        'src/utils/**',
+        'src/backend/utils/view-helpers/view-helpers.ts',
+        'src/backend/utils/filter/filter.ts',
+        'src/backend/decorators/**',
       ],
     },
   })
 }
 
 build().catch((error) => {
   console.log(error)
+  process.exit(1)
```

Comment:
With this the process crash with a error code different from 0 and then make the action fail.